alma/tests/unit/lib/API/EligibilityHelperTest.php

```diff
@@ -0,0 +1,191 @@
+     * @return array[]
+     */
+    public function getEligibilityData() {
+        $dataAmountEligibe = [
+            'eligible' => true,
+            'reasons' => NULL,
+            'constraints' => NULL,
+            'payment_plan' => [
+                [
+                    'customer_fee' => 61,
+                    'customer_interest' => 0,
+                    'due_date' => 1656429874,
+                    'purchase_amount' => 1701,
+                    'total_amount' => 1762,
+                ],
+                [
+                    'customer_fee' => 0,
+                    'customer_interest' => 0,
+                    'due_date' => 1659021874,
+                    'purchase_amount' => 1699,
+                    'total_amount' => 1699,
+                ],
+                [
+                    'customer_fee' => 0,
+                    'customer_interest' => 0,
+                    'due_date' => 1661700274,
+                    'purchase_amount' => 1699,
+                    'total_amount' => 1699,
+                ]
+            ],
+            'installments_count' => 3,
+            'deferred_days' => 0,
+            'deferred_months' => 0,
+            'customer_total_cost_amount' => 61,
+            'customer_total_cost_bps' => 120,
+            'annual_interest_rate' => NULL,
+        ];
+
+        $dataAmountNotEligibe = [
+            'eligible' => false,
+            'reasons' => NULL,
+            'constraints' => [
+                'purchase_amount' => [
+                    'minimum' => 5000,
+                    'maximum' => 300000,
+                ]
+            ],
+            'payment_plan' => NULL,
+            'installments_count' => 3,
+            'deferred_days' => 0,
+            'deferred_months' => 0,
+            'customer_total_cost_amount' => NULL,
+            'customer_total_cost_bps' => NULL,
+            'annual_interest_rate' => NULL,
+        ];
+
+        $expectedAmountEligibe = new Eligibility($dataAmountEligibe);
+        $expectedAmountNotEligibe = new Eligibility($dataAmountNotEligibe);
+
+        return [
+            'purchase amount eligible in p3x' => [
+                //data
+                [
+                    'purchase_amount' => 5099,
+                    'queries' => [
+                        [
+                            'purchase_amount' => 5099,
+                            'installments_count' => 3,
+                            'deferred_days' => 0,
+                            'deferred_months' => 0
+                        ]
+                    ],
+                    'shipping_address' => [
+                        'country' => 'FR'
+                    ],
+                    'billing_address' => [
+                        'country' => 'FR'
+                    ],
+                    'locale' => 'en'
+                ], 
+                //data expected
+                [
+                    $expectedAmountEligibe
+                ]
```

Comment:
```suggestion
                //data expected
                $expectedAmountEligibe
```

it is not necessary to put it in an array (you will able to use it without accessing the array first element in your test ;) `$expected[0]`)

Reply:
@syjust-alma I don't think is more readable by ok is done :)